src/components/Main/SeverityTable/SeverityTable.tsx

```diff
@@ -114,7 +116,10 @@ export function getErrorMessages(t: TFunction, errors?: FormikErrors<AgeGroupRow
     const rowNum = Number.parseInt(i, 10)
     const ageGroup = Object.values(AgeGroup)[rowNum]
 
-    if (err) {
+    if (typeof err === 'string') {
+      return [...result, t(`Error in row "{{ageGroup}}": {{message}}`, { ageGroup, message: err })]
+    }
+    if (typeof err === 'object') {
       const messages = Object.entries(err).map(([column, message]) =>
         t('Error in column "{{column}}", row "{{ageGroup}}": {{message}}', { column, ageGroup, message }),
       )
```

Comment:
Adjusted the errors display logic in severity table: when the new check in yup schema fails, the result is being sent as an array with a single string in it (because the check is attached to a particular object in an array of objects).